invenio/utils/deprecation.py

```diff
@@ -25,20 +25,54 @@
 completely removed in version ``A.(B+2)``.
 
 Example:
+
 - Feature X is mark in version *2.0* as deprecated. The feature will still
   work, but it will issue the silent warning ``RemovedInInvenio22Warning``
   (`PendingDeprecationWarning`).
 - Version *2.1* still contains the feature, but the warning
   ``RemovedInInvenio22Warning`` is now loud by default (`DeprecationWarning`).
 - Version *2.2* will completely remove the feature.
+
+Fast track deprecation
+----------------------
+Poorly tested, non-core and little used features for which alternatives already
+exists in Invenio may opt to use fast track deprecation. If a feature is
+fast track deprecated in version ``A.B`` it will continue to work in version
+``A.B`` but raise loud warnings. It will be completely removed in version
+``A.(B+1)``.
```

Comment:
I asked to clarify the text, because "little used" and "non-core" are not clear by themselves.  There may be some *user-facing feature* that only 1 installation out of 100 uses.  Is this "little used" or not?  Globally yes, it is little used.  Locally no, the feature may be of utmost importance to that particular installation.   Using better wording in the text, with concrete example of *internal features* considered for fast deprecation, such as remote debugger in the flask world, would help to alleviate potential concerns that people may be having WRT fast deprecation of some user-facing features by the majority.